src/javascripts/todo-filter.js

```diff
@@ -39,4 +39,4 @@ function updateFilter (filter) {
   }
 }
 
-export default prepFilters
+module.exports = prepFilters
```

Comment:
Usually a transpiler (like Traceur or Babel) is used to convert ES6 code into appropriate ES5 code which can be understood by Node and Browser environments. Using the build files to test the code in the browser will work. But remember, you cannot unit test code from built files. Now how will unit tests work?